### DISCUSSION TITLE
two site on gunicorn unbutu

### DISCUSSION THREAD

**Discussion by IRIETech225** [benoitc/gunicorn#2981] [General]
hello

i follow this tutorial  (https://www.rosehosting.com/blog/how-to-setup-django-with-postgres-nginx-and-gunicorn-on-ubuntu-20-04/ ) 
for install a django website  on my unbutu test server 

now i search a good configuration for a second website 

here the configuration for the first website that run very well

/etc/systemd/system/gunicorn.service
------------------------------------
[Unit]
Description=gunicorn daemon
After=network.target

[Service]
User=irietech
Group=www-data
WorkingDirectory=/home/irietech/django
ExecStart=/home/irietech/django/djangoenv/bin/gunicorn --access-logfile - --workers 3 --bind unix:/home/irietech/django/branda.sock branda.wsgi:application

[Install]
WantedBy=multi-user.target

--------------------------------
how can i add an other website on this config 

thanks for help a new python dev

new website name is shopin and the it working directory is 

WorkingDirectory=/home/irietech/oscar

**Comment by benoitc** [comment]
I would create a new service for it in systemd and name them accordingly.
